digicampipe/visualization/time_resolution_plot.py

```diff
@@ -154,13 +157,13 @@ def plot_resol(data_file, legend, ax=None):
     mean_charge_all, std_charge_all, mean_t_all, std_t_all, true_pe = \
         load_data(data_file)
     if ax is None:
-        fig, ax = plt.subplots(1, 1, figsize=(8, 6), dpi=100)
+        fig, ax = plt.subplots(1, 1, figsize=(8, 8), dpi=100)
     # ax.plot([20*pde, 2e3*pde], [1, 1], 'r-', label='requirement B-TEL-1380')
     # ax.plot([20*pde, 20*pde], [.9, 1.1], 'r-', label=None)
     ax.plot([20*pde, 2e3*pde], [3, 3], 'b--', label='requirement B-TEL-1640')
```

Comment:
I think ax.axhline(3) would be better to use. same for the following plots

Reply:
only annoying thing with ax.axhline() is that the xmin and xmax have to be given in relative coordinates (0 to 1)...